groups/bsl/bsls/bsls_spinlock.h

```diff
@@ -292,7 +292,7 @@ extern "C" {
 #include <time.h>
 #endif
 
-#if !(defined(BSLS_PLATFORM_OS_SOLARIS)) && !(defined(BSLS_PLATFORM_OS_AIX))
+#if !(defined(BSLS_PLATFORM_OS_SOLARIS)) && !(defined(BSLS_PLATFORM_OS_AIX)) && !(defined(__arm64))
```

Comment:
Wordwrap with a line continuation for exceeding 79 columns

Reply:
We shouldn't reference system macros directly but should be checking for the proper BSLS_PLATFORM macros.

Please change this (and the other preprocessor check below) to this:

```
#if    !(defined(BSLS_PLATFORM_OS_SOLARIS))                                   \
    && !(defined(BSLS_PLATFORM_OS_AIX))                                       \
    && !(defined(BSLS_PLATFORM_OS_DARWIN) && defined(BSLS_PLATFORM_CPU_ARM))
```